src/foam/dao/Relationship.js

```diff
@@ -254,18 +257,21 @@ foam.CLASS({
         foam.RELATIONSHIP({
           sourceModel: this.targetModel,
           targetModel: id,
+          targetProperty: this.targetProperty,
           forwardName: this.inverseName,
           inverseName: 'targetId',
           sourceDAOKey: this.targetDAOKey,
           targetDAOKey: this.junctionDAOKey,
-          relationshipDAOFactory: function(source) {
+          relationshipDAOFactory: function(s) {
             return foam.dao.ManyToManyRelationshipDAO.create({
-              obj: source,
+              dest: jModel,
+              obj: s,
+              of: source.id,
```

Comment:
Oh meant to ask about this. So in the previous code, the source var from the method is clobbered by the source var that's passed into this function and I wasn't sure if I should be using that source.